config.yaml

```diff
@@ -30,6 +30,10 @@ source_config:
 # remove to disable S3 uploads
 s3_bucket: fcos-builds
 
+registry_repos:
+  oscontainer: quay.io/fedora/fedora-coreos
+  old_oscontainer: quay.io/coreos-assembler/fcos
```

Comment:
Given that the old location is a temporary thing, it seems odd to make this a full-blown object. Also, "old oscontainer" might be confused to mean the old oscontainer format.

WDYT about simply:

```yaml
oscontainer_repo: quay.io/fedora/fedora-coreos
oscontainer_old_repo: quay.io/coreos-assembler/fcos
```

And then we can eventually drop support for `oscontainer_old_repo`.

Reply:
OK that makes sense. Re. name, maybe just `oscontainer_old`?

Reply:
Done